tests/scripts/github-action-helper.sh

```diff
@@ -679,6 +679,26 @@ function test_csi_nfs_workload {
   kubectl exec -t pod/csinfs-demo-pod -- ls -alh /var/lib/www/html/
 }
 
+function remove_gcloud_in_runner() {
```

Comment:
We will be able to remove this workaround in the future?

Reply:
yes